src/components/custom/signup.tsx

```diff
@@ -203,26 +220,16 @@ const Signup = (props: SignupProps) => {
                 return <SignupDefault {...parameters}></SignupDefault>
         }
     }
+
     return props.submit_state === 'success' ? (
         <ResponseWrapper>
-            <Header as="h3" type="section-title" align="center" weight="normal">
+            <Header as="h3" type="subtitle-1" align="center" weight="700">
```

Comment:
Header's weight == bold == 700 by default

Reply:
Thanks for your comments @NikitK-deriv , fixed everything in new commit.